5Homework18.03.22/Routers/Routers.Tests/ConfigurationGeneratorTests.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.IO;
+using NUnit.Framework;
+
+namespace Routers.Tests;
+
+public class Tests
+{
+    private bool FileCompare(string path1, string path2)
+    {
+        const int BYTES_TO_READ = sizeof(Int64);
+        FileInfo first = new FileInfo(path1);
+        FileInfo second = new FileInfo(path2);
+        if (first.Length != second.Length)
+            return false;
+
+        if (string.Equals(first.FullName, second.FullName, StringComparison.OrdinalIgnoreCase))
+            return true;
+
+        int iterations = (int)Math.Ceiling((double)first.Length / BYTES_TO_READ);
+
+        using (FileStream fs1 = first.OpenRead())
+        using (FileStream fs2 = second.OpenRead())
+        {
+            byte[] one = new byte[BYTES_TO_READ];
+            byte[] two = new byte[BYTES_TO_READ];
+
+            for (int i = 0; i < iterations; i++)
+            {
+                fs1.Read(one, 0, BYTES_TO_READ);
+                fs2.Read(two, 0, BYTES_TO_READ);
+
+                if (BitConverter.ToInt64(one,0) != BitConverter.ToInt64(two,0))
+                    return false;
+            }
+        }
+
+        return true;
+    }
+    
+    [Test]
+    public void ConfigurationFromTheTask()
+    {
+        IConfigurationGenerator generator = new ConfigurationGenerator("../../../InputFile1.txt");
```

Comment:
```suggestion
        var generator = new ConfigurationGenerator("../../../InputFile1.txt");
```
Вам же не важно, что это должен быть интерфейс